src/installer/installer.go

```diff
@@ -453,6 +456,30 @@ func (i *installer) startBootstrap() error {
 		return err
 	}
 
+	// If we're in a pure RHEL/CentOS environment, we need to overlay the node image
```

Comment:
What happens if we are not in a pure RHEL/Centos environment?
Is this a possible scenario. What should we do in those cases?